packages/framework/src/Model/Seo/HreflangLinksFacade.php

```diff
@@ -100,6 +102,16 @@ public function getForBlogArticle(BlogArticle $blogArticle, int $currentDomainId
         return $this->doGetHrefLinks($blogArticle, $currentDomainId, static::ROUTE_BLOG_ARTICLE_DETAIL);
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Blog\Category\BlogCategory $blogCategory
+     * @param int $currentDomainId
+     * @return \Shopsys\FrameworkBundle\Model\Seo\HreflangLink[]
+     */
+    public function getForBlogCategory(BlogCategory $blogCategory, int $currentDomainId): array
+    {
+        return $this->doGetHrefLinks($blogCategory, $currentDomainId, static::ROUTE_BLOG_CATEGORY_DETAIL);
```

Comment:
blog category also has a visibility that should be probably taken into account here

Reply:
👍 👍  – 64eb7e37fa